addons/website/models/ir_model.py

```diff
@@ -38,3 +39,9 @@ def get_website_meta(self):
         # dummy version of 'get_website_meta' above; this is a graceful fallback
         # for models that don't inherit from 'website.seo.metadata'
         return {}
+
+    def _get_base_lang(self):
+        """ Returns the default language of the website as the base language if the record is bound to it """
```

Comment:
72*

Reply:
72 is just a rule for git messages, because of the 2*4 char padding that tools like `git` and `tig` are adding to reach the 80 char line on terminal.
For docstring, 80 char line is fine.
Tig:
![image](https://github.com/odoo/odoo/assets/30048408/231c3777-d4c0-49fd-9521-cbc4ffed6470)
Git:
![image](https://github.com/odoo/odoo/assets/30048408/f3898a0f-cfdb-4cef-831c-11ca8e5968c2)

So it's 80, not 72 for the diff itself.
Obviously, for line of codes, it's impossible to respect, but all of this I summarized on https://github.com/odoo/odoo/pull/83508#discussion_r796442104